src/components/main/MainCard.tsx

```diff
@@ -21,7 +21,7 @@ const convertTime = (timeData: string) => {
 };
 
 const MainCard: FC<MainCardProps> = ({ currentWeather, location }) => {
```

Comment:
not required to be fixed in current PR, but would be better to use destructuring for `currentWeather` prop in this case